src/Tools/dotnet-monitor/Egress/Extension/EgressExtension.cs

```diff
@@ -148,8 +145,13 @@ public void AddEnvironmentVariable(string key, string value)
 
             parser.BeginReading();
 
-            await JsonSerializer.SerializeAsync<ExtensionEgressPayload>(p.StandardInput.BaseStream, payload, options: null, token);
-            await p.StandardInput.WriteAsync(NewLine, token);
+            Stream intermediateStream = new MemoryStream();
+            await JsonSerializer.SerializeAsync(intermediateStream, payload, options: null, token);
```

Comment:
What is the default pretty printing for serialization? We can save size by making sure the json is as compact as possible.

Reply:
It appears that it's compacted by default (there's no spacing) - I believe there's an option to make it indented, but by default it isn't.